### ISSUE TITLE
Improve tooltip hover interaction in properties panel

### ISSUE THREAD

**Issue by wollefitz** [camunda/camunda-modeler#5217]
### Problem you would like to solve

When hovering over a section title in the properties panel and a tooltip appears, the cursor must be very precisely positioned over it. Any slight movement away from the tooltip causes it to close immediately, making it difficult to interact with it, for example, if I would like to copy the text. 

https://github.com/user-attachments/assets/1373ece7-d048-4b2c-865c-e9a84203ffd7

### Proposed solution

Consider adding a short delay and/or expanding the hover area to prevent accidental closing.

### Alternatives considered

-

### Additional context

This issue is a transferred feature request from @YanaSegal (https://github.com/camunda/web-modeler/issues/15016)

---

Related to https://github.com/camunda/camunda-modeler/issues/4857

**Comment by nikku** [comment]
As a user you want to be able to move over to the tooltip without accidentally loosing focus (and the tooltip is closing).

I added `ready`, because this seems like a violation of [do the hard work to make it simple](https://github.com/bpmn-io/design-principles?tab=readme-ov-file#do-the-hard-work-to-make-it-simple). Our tools should not be cumbersome and hard to use.

Also attached `a11y` label. Not everyone has expert mouse pointing skills.

**Comment by lmbateman** [comment]
Will this address https://github.com/camunda/camunda-modeler/issues/4857?

**Comment by nikku** [comment]
No, but I marked both as related / linked. I believe we should tackle both in one sweep.

**Comment by Buckwich** [comment]
I tried increasing the hover area, but it introduces a whole bunch of other issues with the positioning and content flow. We don't have much play with increasing the size anyway, as we can't/shouldn't go over the input field itself. The marked areas are already considered hover areas: 

<img width="689" height="196" alt="Image" src="https://github.com/user-attachments/assets/2f608e7d-4e65-459c-b3a1-657322f5df4a" />

We could go a bit further to the left (only relevant if tooltip not already shown). on the right we are constrained by the feel button 

<img width="209" height="119" alt="Image" src="https://github.com/user-attachments/assets/e6a89511-1aa2-415d-bd77-75caf2508567" />

If we go with just a closing delay (and handling selections) should be enough
